docs/core/compatibility/serialization/9.0/binaryformatter-removal.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Breaking change: In-box BinaryFormatter is unusable"
```

Comment:
This sounds more like a problem with the quality of the user experience, rather than a deliberate design decision.

```suggestion
title: "Breaking change: In-box BinaryFormatter implementation was removed and always throws"
```